tests/framework/filters/stubs/UserIdentity.php

```diff
@@ -63,4 +64,19 @@ public function validateAuthKey($authKey)
     {
         throw new NotSupportedException();
     }
+
+    public function getRateLimit($request, $action)
```

Comment:
In this case, the api will be broken.
The programmer must implement the RatLimitInterface for the use of RateLimiter.
The structure is reliable than the array.